test/TestUtils.py

```diff
@@ -258,3 +259,28 @@ def mockreturn(options):
     monkeypatch.setattr(utils, 'get_yaml_files', mockreturn)
     result = utils.get_playbooks_and_roles(options)
     assert result == ['bar/playbook.yml']
+
+
+def test_get_rules_dirs():
+    default = [constants.DEFAULT_RULESDIR]
+    uruledirs = ["/tmp/rules/custom/99", "/tmp/rules/custom/10"]
+
+    assert utils.get_rules_dirs([], True) == default
+    assert utils.get_rules_dirs([], False) == default
+    assert utils.get_rules_dirs(uruledirs, True) == uruledirs + default
+    assert utils.get_rules_dirs(uruledirs, False) == uruledirs
```

Comment:
Tests should have one `assert` statement. Use `@pytest.mark.parametrize` to run the same test with different params.

Reply:
Thanks a lot for your advice! I tried that in the latest commit. Please let me know if I'm doing somethng wrong because I'm still not familiar with pytest.

Reply:
@ssato great! I've posted some pytest pointers @ https://github.com/ansible/ansible-lint/issues/725 but feel free to also ask questions about pytest if you have any.